src/service/platform-impl.js

```diff
@@ -79,7 +79,8 @@ export class Platform {
    */
   isIe() {
     return (/MSIE/i.test(this.navigator_.userAgent) ||
```

Comment:
Hoist this function out, import it in the polyfill. Also, let's join the regexes.